fli-iam/shanoir-ng
# #2256-missing-mr-sequence-variant

Closing #2256 

To avoid further Errors, return UNKNOWN when not found

How to test (not that simple)
- Open a dicom with pydicom and update mr-sequence-variant field to set something not in the list
- Import this data
--> Check that this metadata is set to UNKNOWN in shanoir
